kolibri/core/errorreports/apps.py

```diff
@@ -0,0 +1,10 @@
+from django.apps import AppConfig
+
+
+class KolibriErrorConfig(AppConfig):
+    name = "kolibri.core.errorreports"
+    label = "errorreports"
+    verbose_name = "Kolibri ErrorReports"
+
+    def ready(self):
```

Comment:
There's probably no use case for this method. However if there is, then its worth changing its body from `...` to `pass` that is more explicit

Reply:
I thought both were the same thing!

Reply:
Changed to pass!

Reply:
Pass is more explicit. ... could mean a host of things.
It goes back to my earlier question. Do we really need the ready method?

Reply:
Not now, some apps inside core had it defined this way with pass. So I just copy-pasted

Reply:
I thinks it's best we remove it as it won't really be necessary.